ingestion/examples/workflows/glue.yaml

```diff
@@ -9,7 +9,6 @@ source:
         awsSecretAccessKey: aws secret access key
         awsRegion: aws region
         endPointURL: https://glue.<region_name>.amazonaws.com/
-      database: local_glue_db
       storageServiceName: storage_name
       pipelineServiceName: pipeline_name
```

Comment:
Removing the pipeline ingestion should also be translated in the JSON Schema of the Glue connector. We could remove the `pipelineServiceName` prop

Reply:
Sure.